src/Microsoft.ML.Data/Transforms/doc.xml

```diff
@@ -9,10 +9,10 @@
         This transform removes the entire row if any of the input columns have a missing value in that row.
         This preprocessing is required for many ML algorithms that cannot work with missing values.
         Useful if any missing entry invalidates the entire row.
-        If the <see cref="Microsoft.ML.Runtime.Data.NAFilter.Defaults.Complement"/> is set to true, this transform would do the exact opposite,
```

Comment:
>Complement [](start = 70, length = 10)

XML comment has cref attribute 'Complement' that could not be resolved

Reply:
added the P: notation in front of the namespace. 
@sharwell, this makes the compiler happy 'sometimes'. 

---
In reply to: [208667529](https://github.com/dotnet/machinelearning/pull/665#discussion_r208667529) [](ancestors = 208667529)

Reply:
Is that the right class name? Found this instead: microsoft.ml.runtime.data.nafilter.**arguments**.complement

Reply:
they are both the same thing, but using this one since the Microsoft.ML namespace is the the user facing one with the current api. 
they will eventually get reconciled with the new api work. 

---
In reply to: [208697780](https://github.com/dotnet/machinelearning/pull/665#discussion_r208697780) [](ancestors = 208697780)

Reply:
Defaults.Complement isn't showing in the API browser

Reply:
[https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.missingvaluesrowdropper.complement?view=ml-dotnet](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.missingvaluesrowdropper.complement?view=ml-dotnet)

---
In reply to: [208699506](https://github.com/dotnet/machinelearning/pull/665#discussion_r208699506) [](ancestors = 208699506)

Reply:
Ahh, sorry, Didn't see the update.